api.web.yaml

```diff
@@ -134,40 +141,42 @@ paths:
         default:
           description: Unexpected error
 
-  /people:
+  /users:
```

Comment:
also need to update nav with the url, do we want to call the page Users instead of People as well? Or do we think for researchers keeping People on the UI makes more sense?

Reply:
eh, let's leave it as people and see if it needs to be updated after user testing then